.dockerignore

```diff
@@ -1,2 +1,8 @@
+.dockerignore
+Dockerfile
 .dart_tool/
-pubspec.lock
+.git/
+.github/
+.gitignore
+.idea/
+.packages
```

Comment:
can drop this one. Not been created for a while!

Reply:
Actually, `dart pub get` still creates it 🤷‍♂️  I just tried deleting it and it got recreated by `dart pub get`. So I think we should keep it for now.